Add `The Followpoint: Dreamxiety, the osu! Artist Extraordinaire` news post (#12058)
ppy/osu-wiki: news/2024/2024-09-02-the-followpoint-dreamxiety-the-osu-artist-extraordinaire.md

---
layout: post
title: "The Followpoint: Dreamxiety, the osu! Artist Extraordinaire"
date: 2024-09-02 12:00:00 +0000
series: The Followpoint
---

> The Followpoint has came back with another interview! This time it will be very artistic you could say.

Reviewer: ```suggestion
The Followpoint has came back with another interview! This time it will be very artistic so to say.
```

![](/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-osu-artist-extraordinaire/banner.jpg)

> What's this? A new interview from The Followpoint? That's right, today we are back with [Dreamxiety](https://osu.ppy.sh/users/13103233). You may know her for creating artworks for the seasonal art contests, and you've probably seen them in the main menu whenever you load up osu!, if you even have seasonal backgrounds enabled... why do you guys have them disabled?

Reviewer: ```suggestion
What's this? A new interview from The Followpoint? That's right, today we are back with [Dreamxiety](https://osu.ppy.sh/users/13103233). You may know her for creating artworks for the seasonal art contests, and you've probably seen them in the main menu whenever you load up osu!, if you have them enabled that is...
```

> Let's get right into it!

Reviewer: ```suggestion
Anyway, let's get right into it!
```
